lib/puppet/type/f5_persistencecookie.rb

```diff
@@ -0,0 +1,53 @@
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/parameter/f5_name.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_address.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_availability_requirement.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_connection_limit.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_connection_rate_limit.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_description.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_health_monitors.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_ratio.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_state.rb'))
+
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_truthy.rb'))
+
+
+Puppet::Type.newtype(:f5_persistencecookie) do
+  @doc = 'Manage cookie persistence objects'
+
+  apply_to_device
+  ensurable
+
+  newparam(:name, :parent => Puppet::Parameter::F5Name, :namevar => true)
+
+  newproperty(:description, :parent => Puppet::Property::F5Description)
+
+  newproperty(:method) do
+    desc "Fail Safe Action. Valid values are 'reboot' or 'restart-all'."
+    newvalues(:'insert', :'passive', 'rewrite')
```

Comment:
Are these strings to symbols?

Reply:
strings

Reply:
I agree with @alexjfisher in that you @ericzji should make all of these symbols (i.e., turn 'rewrite' into :rewrite)